src/Symfony/Component/Workflow/Tests/StateMachineTest.php

```diff
@@ -38,4 +41,76 @@ public function testCanWithMultipleTransition()
         $this->assertTrue($net->can($subject, 't2'));
         $this->assertTrue($net->can($subject, 't3'));
     }
+
+    public function testBuildTransitionBlockerList()
+    {
+        $definition = $this->createComplexStateMachineDefinition();
+
+        $net = new StateMachine($definition);
+        $subject = new \stdClass();
+
+        // If you are in place "a" you should be able to apply "t1"
```

Comment:
not sure this adds that much value, also it may be confusing why the comment is only here but below for `d` and `b`